Code/Legacy/CrySystem/SystemInit.cpp

```diff
@@ -993,14 +993,19 @@ bool CSystem::Init(const SSystemInitParams& startupParams)
         }
 
         //////////////////////////////////////////////////////////////////////////
+#if defined(CARBONATED)
+        if (Legacy::System::CVars::sys_asserts == 0)
+#else
         if (g_cvars.sys_asserts == 0)
+#endif
         {
             gEnv->bIgnoreAllAsserts = true;
         }
+#if defined(CARBONATED)
+        if (Legacy::System::CVars::sys_asserts == 2)
+#else
         if (g_cvars.sys_asserts == 2)
-        {
-            gEnv->bNoAssertDialog = true;
-        }
```

Comment:
Why were these lines removed?

If CARBONATED was not defined, this would result to:
```
if (g_cvars.sys_asserts == 2)
LogBuildInfo();
```